scripts/needs_ci_run

```diff
@@ -1,12 +1,11 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
```

Comment:
hmpf, took me longer than it should've to understand what was happening.. in circleci we have to use `sudo` to install the dependencies (also python3-requests). it uses `/usr/bin/python3` for that. this script is being run without `sudo` and references `/opt/circleci/.pyenv/versions/3.10.5/bin/python3` inside circleci's virtual env and apparently they didn't bother to include `/usr/lib/python3.10/site-packages` in the PYTHONPATH, so importing `requests` fails.. 

since it only runs in ci I think it's fair to use this line instead